monitoring/configure-monitoring-wds.sh

```diff
@@ -58,6 +58,11 @@ SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
 # set core context
 kubectl config use-context $ctx
 
+: --------------------------------------------------------------------
+: Configure APF for prometheus traffic to the WDS API server
+: --------------------------------------------------------------------
+kubectl --context $wds apply -f ${SCRIPT_DIR}/configuration/prometheus-pod-exempt.yaml
```

Comment:
I just ran `test/e2e/run-test.sh`; `cd monitoring`; `./install-ks-monitoring.sh`. I find that there is a ServiceAccount named `prometheus-kube-prometheus-prometheus` in a namespace named `ks-monitoring` in the KubeFlex hosting cluster and NOT in the WDS.

Reply:
After proceeding further through the instructions, I see that there now _is_ a ServiceAccount named `prometheus-kube-prometheus-prometheus` in a namespace named `ks-monitoring` in the WDS.

I think that we want the `prometheus-exempt` FlowSchema to exist in the WDS and in the KubeFlex hosting cluster.

Reply:
And the ITS too.